### ISSUE TITLE
Advice in README needs a wee update

### ISSUE THREAD

**Issue by paul-hammant** [micro-frontends-demo/container#41]
README currently says "You can run the container on its own", but I think it needs 'content' running (default port 5000) or else there's an immediate can't find React error. 

After that though, there's a missing 'http://localhost:5000/restaurants.json' resource (same port) and now I'm confused to whether I'm correct with the above suggested advise.

Maybe shouldn't be looking at the chrome tab that 'yarn start' itself opens at all.

**Comment by maxgabut** [comment]
I think you're right about 'container' needing 'content'. It's the later that serves React to the former.

Regarding `restaurant.json`, it's necessary to run `yarn build` in the 'content' project to generate it.

**Comment by camjackson** [comment]
Yes you're correct, the content app needs to be running for anything to be functional. The readme does say:

> You can run the container on its own, but for it to actually do anything you'll also need to be running:
> - the [content](https://github.com/micro-frontends-demo/content) server
> - the [browse](https://github.com/micro-frontends-demo/browse/) micro frontend
> - the [restaurant-order](https://github.com/micro-frontends-demo/restaurant-order) micro frontend

Happy to take a PR if anyone has a better suggested wording.
